src/Apps/Order/Components/CheckoutAddress.tsx

```diff
@@ -48,19 +72,126 @@ const VALIDATION_SCHEMA = Yup.object().shape({
   }),
 })
 
-const getDefaultCountry = (userCountry: string): string => {
-  const countryCode = ALL_COUNTRY_SELECT_OPTIONS.find(
-    country => country.text === userCountry
-  )?.value
-  return countryCode || "US"
-}
-
 export const CheckoutAddress: FC<{
   userCountry: string
   onChange: AddressChangeHandler
 }> = ({ userCountry, onChange }) => {
-  const { submitMutation: submitAddAddress } = useAddAddress()
-  const userDefaultCountry = getDefaultCountry(userCountry)
+  const userDefaultCountry = getDefaultCountry(userCountry, true)
+  const [displayModal, setDisplayModal] = useState(false)
+  const [modalTitle, setModalTitle] = useState<ErrorModalTitle>(
+    ErrorModalTitle.general
+  )
+  const [modalContent, setModalContent] = useState<ReactElement>()
+  const [selectedAddress, setSelectedAddress] = useState<
+    AddressSuggestionRadioButton
+  >(AddressSuggestionRadioButton.recommended)
+
+  const handleKeepAddressClick = () => {}
+
+  const handleEditAddressClick = () => {
+    setDisplayModal(false)
+  }
+
+  const handleUseSuggestedAddressClick = () => {}
+
+  const handleFormSubmit = async (address: Address) => {
+    const { firstLine, secondLine } = formatAddress(address)
+
+    const res = await mockRequest()
+    switch (res) {
+      case 0:
+        setModalTitle(ErrorModalTitle.general)
+        setModalContent(
+          <>
+            <Spacer y={2} />
+            <Text>
+              The address you entered may be incorrect or incomplete. Please
+              check it and make any changes necessary.
+            </Text>
+            <Spacer y={4} />
+            <Text fontWeight={600}>What you entered</Text>
+            <Spacer y={1} />
+            <Box border="1px solid #C2C2C2" p={20}>
+              <Text>{firstLine}</Text>
+              <Text>{secondLine}</Text>
+            </Box>
+            <Spacer y={4} />
+            <Flex width="100%" justifyContent="space-between">
+              <Button
+                onClick={handleKeepAddressClick}
+                variant="secondaryBlack"
+                width="49%"
+              >
+                Use This Address
+              </Button>
+              <Button onClick={handleEditAddressClick} width="49%">
```

Comment:
Don't use percentages like this; use a flex along with a spacer

Reply:
pushing an update 🙏